libs/build_yafyaml.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -eux
+
+name="yafyaml"
+repo=${1:-${STACK_yafyaml_repo:-"Goddard-Fortran-Ecosystem"}}
+version=${2:-${STACK_yafyaml_version:-"main"}}
+
+# Hyphenated version used for install prefix
+compiler=$(echo $HPC_COMPILER | sed 's/\//-/g')
+id=${version//\//-}
+
+if $MODULES; then
+  set +x
+  source $MODULESHOME/init/bash
+  module load hpc-$HPC_COMPILER
+  module try-load cmake
+  module load gftl-shared
+  module list
+  set -x
+
+  prefix="${PREFIX:-"/opt/modules"}/$compiler/$name/$repo-$id"
+  if [[ -d $prefix ]]; then
+    [[ $OVERWRITE =~ [yYtT] ]] && ( echo "WARNING: $prefix EXISTS: OVERWRITING!"; $SUDO rm -rf $prefix; $SUDO mkdir $prefix ) \
+                               || ( echo "WARNING: $prefix EXISTS, SKIPPING"; exit 1 )
+  fi
+else
+  prefix=${CMAKEMODULES_ROOT:-"/usr/local"}
```

Comment:
This is repeatedly incorrect.

Reply:
Addressed in 2c049520a84a6938bc1724058b4e2d81c30376ae